src/tagproInterface.js

```diff
@@ -17,6 +15,43 @@ export function chat(chatMessage) {
   }
 }
 
+let AUTONOMOUS = true;
+let VISUALS = true;
+
+export function isAutonomous() {
+  return AUTONOMOUS;
+}
+
+export function visualMode() {
+  return VISUALS;
+}
+
+export function onKeyDown(event) {
+  // If letter pressed is Q, toggle autonomous controls
+  if (event.keyCode === 81) {
+    AUTONOMOUS = !AUTONOMOUS;
+    VISUALS = AUTONOMOUS;
+    tagpro.sendKeyPress('up', true);
+    tagpro.sendKeyPress('down', true);
+    tagpro.sendKeyPress('left', true);
+    tagpro.sendKeyPress('right', true);
+    if (AUTONOMOUS) {
+      chat('Autonomy Mode updated: now AUTONOMOUS!');
+    } else {
+      chat('Autonomy Mode updated: now MANUAL!');
+    }
+    setTimeout(() => { console.log(`Autonomy status: ${AUTONOMOUS}`); }, 200);
+  }
+  if (event.keyCode === 86) { // 'v'
```

Comment:
Do we want to give the keycodes that we use variable names? That way we won't have a random `81` and `86`?